documentation/modules/configuring/proc-manual-restart-mirrormaker2-connector-task.adoc

```diff
@@ -2,45 +2,43 @@
 // assembly-management-tasks.adoc
 
 [id='proc-manual-restart-mirrormaker2-connector-task-{context}']
-= Performing restarts of MirrorMaker 2 connector tasks using annotations
+= Manually restarting MirrorMaker 2 connector tasks
 
-This procedure describes how to manually trigger a restart of a Kafka MirrorMaker 2 connector task by using a Kubernetes annotation.
+[role="_abstract"]
+Use the `restart-connector-task` annotation to manually trigger a restart of a MirrorMaker 2 connector.
 
 .Prerequisites
 
 * The Cluster Operator is running.
 
 .Procedure
 
-. Find the name of the `KafkaMirrorMaker2` custom resource that controls the Kafka MirrorMaker 2 connector you want to restart:
+. Find the name of the `KafkaMirrorMaker2` custom resource that controls the MirrorMaker 2 connector task you want to restart:
 +
 [source,shell,subs="+quotes"]
 ----
 kubectl get KafkaMirrorMaker2
 ----
 
-. Find the name of the Kafka MirrorMaker 2 connector and the ID of the task to be restarted from the `KafkaMirrorMaker2` custom resource.
-Task IDs are non-negative integers, starting from 0.
+. Find the name of the connector and the ID of the task to be restarted from the `KafkaMirrorMaker2` custom resource:
 +
 [source,shell,subs="+quotes"]
 ----
-kubectl describe KafkaMirrorMaker2 _KAFKAMIRRORMAKER-2-NAME_
+kubectl describe KafkaMirrorMaker2 <mirrormaker_cluster_name>
 ----
++
+Task IDs are non-negative integers, starting from 0.
 
-. To restart the connector task, annotate the `KafkaMirrorMaker2` resource in Kubernetes.
-In this example, `kubectl annotate` restarts task _0_ of a connector named `+my-source->my-target.MirrorSourceConnector+`:
+. Use the name and ID to restart the connector task by annotating the `KafkaMirrorMaker2` resource in Kubernetes:
 +
 [source,shell,subs="+quotes"]
 ----
-kubectl annotate KafkaMirrorMaker2 _KAFKAMIRRORMAKER-2-NAME_ "strimzi.io/restart-connector-task=my-source->my-target.MirrorSourceConnector:0"
+kubectl annotate KafkaMirrorMaker2 <mirrormaker_cluster_name> "strimzi.io/restart-connector-task=<mirrormaker_connector_name>:0"
```

Comment:
I wonder if having an example also with the actual name would be helpful given the name is long and complicated.

Reply:
Yes, we can.

```
In this example, task `0` for connector `my-connector` in the `my-mirror-maker-2` cluster is restarted:
+
[source,shell,subs="+quotes"]
----
kubectl annotate KafkaMirrorMaker2 my-mirror-maker-2 "strimzi.io/restart-connector-task=my-connector:0"
```